examples/bullet.usc

```diff
@@ -0,0 +1,37 @@
+local bd = require("blockdiagram")
+
+return bd.system
+{
+   -- import microblx modules
+   imports = {
+      "stdtypes", "ptrig", "lfds_cyclic", "sim2b_bullet",
+   },
+
+   -- describe which blocks to instantiate
+   blocks = {
+      { name="sim", type="sim2b/sim_bullet" },
+      { name="trig1", type="ubx/ptrig" }
+   },
+
+   -- connect blocks
+   connections = {
+   },
+
+   -- configure blocks
+   configurations = {
+      { name="sim", config = { nr_joints=7,
+                               gravity={0,0,-9.81},
+                               jnt_pos_init={1,2,3,4,5,6,7},
+                               urdf="/tmp/2b-makefile/vericomp-application@master/kortex_description/arms/gen3/7dof/urdf/gen3.urdf" } },
```

Comment:
Could we have the `2dof.urdf` robot from the stand-alone example also here? Then it was possible to directly run this example script.

Reply:
I can change the degrees of freedom and the urdf file, but what should the directory layout be like?

Reply:
We could move the `2dof.urdf` example file to the repository root (e.g. `/sim2b/misc/2dof.urdf`) and then copy it to the build root like in the stand-alone example (cf. https://github.com/rosym-project/sim2b/blob/master/src/example/CMakeLists.txt#L5). Do you think that's reasonable?

I also just noticed that the `examples` folder is in the repository root, too. For consistency, I would prefer to have all framework-related items in one place (see https://github.com/rosym-project/robif2b/tree/master/src/ubx/2dof for an example).